### ISSUE TITLE
Don't use <> around "translate with synonyms"

### ISSUE THREAD

**Issue by M1dgard** [openstreetmap/iD#4326]
Transifex now recognizes it as a tag. This has two major problems:

1) It causes some people in my locale to add this "tag" to the translation.
2) It makes it cumbersome to see what you have to translate (you have to hover it to see the original text).

![Screenshot of the Transifex UI, with a wrong translation in the history panel](https://user-images.githubusercontent.com/4283376/30222355-34ed76a4-94c7-11e7-8335-6c57e1450e71.png)

**Comment by M1dgard** [comment]
Sadly I can't find a way to disable this tag recognition on Transifex (in a project I administer).

**Comment by M1dgard** [comment]
Oh no, I just realized that editing all the source strings will cause the translations to be invalidated...
Perhaps this could be worked around by downloading all translated resources, then doing the change, and then reuploading the translations to Transifex. Or you could contact Transifex to ask to disable tag recognition.

**Comment by bhousel** [comment]
> Oh no, I just realized that editing all the source strings will cause the translations to be invalidated...
> Perhaps this could be worked around by downloading all translated resources, then doing the change, and then reuploading the translations to Transifex. Or you could contact Transifex to ask to disable tag recognition.

Yes - this is a really tricky issue, and unfortunate that the `terms` source strings are now appearing like this.  We might be able to solve it like you said with a shell script that restores the translations after we change the source string.  

I've always thought it was kind of weird that we don't just send the terms list directly as a source strings, and instead we stick an instructions text there.  So maybe if we ever find a way to change this, we can start sending source strings for terms, and put the instructions in the instructions section where it belongs.

**Comment by maraf24** [comment]
@M1dgard, you need to enable raw editor mode in "Settings" menu.

**Comment by M1dgard** [comment]
That works well, but it is not obvious to the other translators that copy the tag.

**Comment by 1ec5** [comment]
> I've always thought it was kind of weird that we don't just send the terms list directly as a source strings, and instead we stick an instructions text there. So maybe if we ever find a way to change this, we can start sending source strings for terms, and put the instructions in the instructions section where it belongs.

Yes, it would make sense to make the instructions part of the context instead of the original string. Apparently [Transifex’s YAML reader](https://docs.transifex.com/formats/yaml) can turn comments into context automatically.

(By the way, please continue to send over the list of terms as a single string. If iD starts sending Transifex a series of separate strings, one for each term, Transifex will require each localization to have the same number of synonyms for “amusement park” (for instance) that the English localization has.)

**Comment by M1dgard** [comment]
To be honest, as a translator, I think the "translate with synonyms..." is better. It makes it clearer what you're translating. The terms are not always relevant: often times the destination language has less or more synonyms. To add to that, a lot of presets don't even have terms filled out in English. Just my 2 cents.

**Comment by bhousel** [comment]
>> So maybe if we ever find a way to change this, we can start sending source strings for terms, and put the instructions in the instructions section where it belongs.
>
> By the way, please continue to send over the list of terms as a single string.

@1ec5:  Oh definitely - sorry if it wasn't clear from what I wrote, but we'd definitely still send them as a single string.

@M1dgard:  We can put translation instructions down in the bottom like this - would this be ok?:

<img width="667" alt="screenshot 2017-10-09 17 27 43" src="https://user-images.githubusercontent.com/38784/31359264-3e08b9ca-ad17-11e7-8327-977680b06ef9.png">

**Comment by M1dgard** [comment]
As far as I can see, [this is what you propose](https://user-images.githubusercontent.com/4283376/34448344-ab332afc-ecec-11e7-809a-67365222f90b.png). (Or does Transifex ellipsize long source texts? Hm, I don't know that.)

I don't like it.

* In the list:
    * stuff without terms is just blank;
    * stuff with a lot of terms takes up a lot of space without benefit.
* In the editor:
    * I use `History` while reviewing, and `Suggestions` for new strings. Switching back and forth would be necessary. For wider viewports, context is always displayed though.

Also, I just find it overall *less clear* what one is supposed to enter, compared to the old situation (`<translate ...>` displayed in full).

**Comment by 1ec5** [comment]
> stuff without terms is just blank;

Not necessarily. The source string could be something like `visitor center, visitor centre`, etc., while the context would explain what it’s for. “String Instructions” contains either the comment from the source file or an instruction manually entered into Transifex. But if I’m not mistaken, there’s also a gray bar that appears in the main panel, between the source string and the copy/undo row; this bar, which is much more discoverable, contains the comment from the source file but _not_ the manually entered instruction.

> stuff with a lot of terms takes up a lot of space without benefit.

This is already the case in the left sidebar.
